zenpy/lib/api.py

```diff
@@ -1026,6 +1026,31 @@ def comments(self, ticket):
         """
         return self._query_zendesk(self.endpoint.comments, 'comment', id=ticket)
 
+    def permanently_delete(self, tickets):
+        """
+        Permanently delete ticket - https://developer.zendesk.com/rest_api/docs/support/tickets#delete-ticket-permanently
+
+        :param ticket: Ticket object
+        """
+        endpoint_kwargs = dict()
+        if isinstance(tickets, collections.Iterable):
+            endpoint_kwargs['destroy_ids'] = [i.id for i in tickets]
+        else:
+            endpoint_kwargs['id'] = tickets.id
+        url = self._build_url(self.endpoint.deleted(**endpoint_kwargs))
+        deleted_ticket_job_id = self._delete(url)
+        self.cache.delete(deleted_ticket_job_id)
```

Comment:
This won't remove the ticket from the cache. Better off passing the `tickets` parameter to cache.delete and just assume the job completes successfully. In the worse case we remove a ticket from the cache that is still live, but that's better then leaving a deleted ticket in the cache.